src/demo/ui.jsx

```diff
@@ -626,6 +653,54 @@ class Predict extends React.Component {
 }
 
 class Pond extends React.Component {
+  onPondClick(e) {
+    const state = getState();
+    const clickX = e.nativeEvent.offsetX;
```

Comment:
out of curiosity, why do we get the offsets from nativeEvent here?

Reply:
I found this technique online earlier today but now am not sure where.  It seems to avoid having to do additional offsetting such as in https://stackoverflow.com/a/18053642 or https://stackoverflow.com/a/33063222.  I suspect we'll likely need to do something like the latter to handle a scaled canvas as we firm up this code soon.